aio_pika/exceptions.py

```diff
@@ -11,9 +11,7 @@
 )
 
 
-PAMQP_EXCEPTIONS = (pamqp.exceptions.PAMQPException,) + tuple(
-    pamqp.specification.ERRORS.values(),
-)
+PAMQP_EXCEPTIONS = (pamqp.exceptions.PAMQPException,)
```

Comment:
Can we get rid of this tuple? It was created to collect all pamqp exceptions because it wasn't trivial until now.

Reply:
done